src/pages/MovieInfo.js

```diff
@@ -34,51 +63,56 @@ export function MovieInfo() {
-          </div>
-          <div className="MovieInfo-data___information">
-            <div className="MovieInfo__title">
-              {' '}
-              <h2>{movie.title} </h2>{' '}
-              <Button img={<BsFillBookmarkFill />}> {t('movieDetails.btn.add')} </Button>
-            </div>
-            <p> {movie.description} </p>
-            <div className="MovieInfo__genres">
-              {' '}
-              {movie?.genres.map((genre, i) => {
-                return <div key={i}> {genre} </div>;
-              })}{' '}
-            </div>
-            <div className="MovieInfo__rate">
-              {' '}
-              <div className="MovieInfo__imdbRating">
-                {' '}
-                <h4 className="MovieInfo-rating__title">{t('movieDetails.ratings.imdb')}</h4>{' '}
-                <h4 className="MovieInfo__ratingStar">
-                  {' '}
-                  <AiFillStar size={25} /> {`${movie.imdbRating} /10`}{' '}
-                </h4>
-                <h5>{movie.allRates} </h5>
-              </div>
-              <div className="MovieInfo-display__rating">
-                <h4 className="MovieInfo-rating__title">{t('movieDetails.ratings.user')}</h4>
-                <h4 className="MovieInfo__ownRating">
-                  {' '}
-                  <AiOutlineStar size={25} /> Rate{' '}
-                </h4>
+      <div className="MovieInfo__fullContainer">
+        <div className="MovieInfo__info-container ">
+          <Button customClass="MovieInfo__btn-back" handleClick={() => goBack()}>
+            <BiArrowBack size={20} />
+          </Button>
+          <div className="MovieInfo__data">
+            <div className="MovieInfo-data__container">
+              <div>
+                <img className="MovieInfo__img" src={movie.img} alt={movie.img} />
               </div>
-              <div className="MovieInfo-display__rating">
-                <h4 className="MovieInfo-rating__title">{t('movieDetails.ratings.general')}</h4>
-                <div className="MovieInfo__stadisticRate">
-                  {' '}
-                  <h4> {movie.popularity.showStadistic}</h4>{' '}
-                  <h4> {movie.popularity.changingStadistic} </h4>
-                </div>{' '}
+              <div className="MovieInfo-data___information">
+                <div className="MovieInfo__title">
+                  <h2>{movie.title} </h2>
+                  <Button
+                    handleClick={() => pushMovieWatchlist()}
+                    img={
+                      buttonState ? <BsFillBookmarkFill size={17} /> : <BsBookmarkPlus size={17} />
+                    }
+                  >
+                    {t('movieDetails.btn.add')}
+                  </Button>
+                </div>
+                <p> {movie.description} </p>
+                <div className="MovieInfo__genres">
+                  {movie?.genres.map((genre, i) => {
```

Comment:
Let's try to use more descriptive variables names
```suggestion
                  {movie.genres.map((genre, index) => {
```

Reply:
I don't see this change applied